drivers/flash/flash_stm32_qspi.c

```diff
@@ -657,6 +915,13 @@ static int flash_stm32_qspi_init(const struct device *dev)
 		return ret;
 	}
 
+#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
+	ret = flash_stm32_qspi_gpio_reset(dev);
+	if (ret < 0) {
+		LOG_ERR("QSPI GPIO reset failed (%d)", ret);
+		return ret;
+	}
```

Comment:
It does not seem that `flash_stm32_qspi_gpio_reset` can ever fail, sot this code is redundant.

Reply:
+1